public/app/routes/routes.ts

```diff
@@ -14,12 +14,24 @@ export function setupAngularRoutes($routeProvider, $locationProvider) {
       reloadOnSearch: false,
       pageClass: 'page-dashboard',
     })
+    .when('/d/:uid/:slug', {
+      templateUrl: 'public/app/partials/dashboard.html',
+      controller: 'LoadDashboardCtrl',
+      reloadOnSearch: false,
+      pageClass: 'page-dashboard',
+    })
     .when('/dashboard/:type/:slug', {
       templateUrl: 'public/app/partials/dashboard.html',
       controller: 'LoadDashboardCtrl',
       reloadOnSearch: false,
       pageClass: 'page-dashboard',
     })
+    .when('/d-solo/:uid/:slug', {
```

Comment:
This route shouldn't be needed since we have a backend redirect if the `/dashboard-solo/:type/:slug` route are requested.

Reply:
My bad. The route is needed for the png rendering route, i.e. `/render/d-solo/<uid>/<slug>`.

Reply:
@torkelo :point_up: